.idems_app/deployments/plh_asset_packs/config.ts

```diff
@@ -0,0 +1,12 @@
+import { extendDeploymentConfig } from "scripts";
+import { supabaseConfig } from "./encrypted/supabaseConfig";
+
+const config = extendDeploymentConfig({ name: "plh_asset_packs", parent: "plh" });
+// override app constants here
+config.app_config.APP_HEADER_DEFAULTS.title = "plh_asset_packs";
+config.app_config.APP_SIDEMENU_DEFAULTS.title = "plh_asset_packs";
+
+config.app_config.ASSET_PACKS.enabled = true
+config.app_config.ASSET_PACKS.supabase = supabaseConfig
```

Comment:
nit(non-blocking)

As supabase would likely be used across multiple features in the future probably better to have its own config property higher up. If we really want to link here, could use `ASSET_PACKS.provider = 'supabase'` or similar if we expected to have support for multiple providers. But for now probably just stick to one and keep simple.